next.config.js

```diff
@@ -1,10 +1,5 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-// @ts-check
```

Comment:
why did we removed this?

Reply:
Was getting a bunch of type errors maybe because of the `withPWA` conflicting with the original types. We can fix this separately though